db/seeds.rb

```diff
@@ -78,9 +78,20 @@ def rugged_repo
   end
 
   puts "Adding Track: #{track_slug}"
+
+  begin
+    git_track = Git::Track.new(track_slug, repo_url:"https://github.com/exercism/v3")
+    title = git_track.config[:language]
+    blurb = git_track.config[:blurb]
+  rescue => e
```

Comment:
This will fail rubocop. 
```suggestion
  rescue StandardError
```